app/views/suggestions/index.html.erb

```diff
@@ -0,0 +1,17 @@
+
+
+
+<div class="container">
+  <%# if @user %>
+    <%#= render partial: 'searchform' %>
+  <%# end %>
+
+  <div class="row">
+    <% @suggestions.each do |suggestion| %>
+      <div class="col-md-6 bordered">
+      <% @suggestion = suggestion %>
+      <%= render partial: 'suggestion', locals: {suggestion: @suggestion} %>
```

Comment:
Really nice job using partials with local variables here! I don't know for sure, but I don't think that you need to set @suggestion as an instance variable here. I believe that you could just pass in the variable suggestion to your locals hash instead of @suggestion.